internal/zinc-compile-core/src/test/scala/sbt/internal/inc/javac/JavaCompilerSpec.scala

```diff
@@ -192,20 +195,43 @@ class JavaCompilerSpec extends UnitSpec with Diagrams {
     lineNumberCheck && messageCheck
   }
 
-  def lineMatches(p: Problem, lineno: Int, lineContent: Option[String] = None): Boolean = {
+  def lineMatches(
+      p: Problem,
+      lineno: Int,
+      colno: Int,
+      lineContent: Option[String] = None
+  ): Boolean = {
     def lineContentCheck = lineContent forall (content => p.position.lineContent contains content)
     def lineNumberCheck = p.position.line.isPresent && (p.position.line.get == lineno)
-    lineNumberCheck && lineContentCheck
+    def columnCheck = {
+      val res = p.position.pointer.isPresent && (p.position.pointer.get == colno)
+      if (!res) {
+        println(s"got ${p.position().pointer().get}, expected $colno, problem $p")
+      }
```

Comment:
I am not sure if println out to stdout is a great idea.

Reply:
Never mind. This is just in the test.

Reply:
Ah, I'm so sorry, I used this for debugging the tests and forgot to remove it. I will open a follow up to remove it.